feat(sdk): Allow user to explicitly call the webview render method. (#139)
hCaptcha/HCaptcha-ios-sdk: README.md

       ...
   }
```

> ### Force the webview Redrawing
> 
> If you want to update the CGRect of the webview displaying the Hcaptcha, (in cases where you may have other conflicting element on your app page), you can call hcaptcha.reload() to froce a redraw without resetting the challenge.

Reviewer: ```suggestion
### Forcing the Webview to Redraw

If you want to update the `CGRect` of the webview displaying the HCaptcha control, e.g. in cases where you may have other conflicting element on your app page, you can call `hcaptcha.reload()` to force a redraw (re-render) without resetting the challenge.
```

```
hcaptcha.validate(on: view) { result in

> 	//You may want to compute a new height for your webview based on the state of the challenge

Reviewer: ```suggestion
	// You may want to compute a new height for your webview based on the state of the challenge
```

    	let newHeight = 100

> 	//reset your webview bounds
> 	placeholder.uiview.bounds = CGRect(x:0,y:0,width:400,height:newHeight)	
> 
> 	//just call this method to explicitly redraw the webview

Reviewer: ```suggestion
	// reset your webview bounds
	placeholder.uiview.bounds = CGRect(x:0,y:0,width:400,height:newHeight)	

	// call this method to explicitly redraw the webview
```

	hcaptcha.reload()

> }

Reviewer 2: @pblanchetteNventive is it exact use case?

Because it looks strange to request on layout update for webview on `validate`'s closure with validation result object

Maybe you can explant the case when it need in more details, why it may need in the middle of `validate` call?

Author: The main reason it's done here in the validate call is because AFAIK there's no HCaptcha event received when the challenge succeeds, otherwise this reload() method could be called in the .onEvent() callback.

Reviewer 2: Thanks for explanation,  the SDK calls validate's result closure once token is generated and usual use case is to 'hide' webView.

To be one the same page you wanna change hCaptcha side and continue show hCaptcha for some time?

Author: > To be one the same page you wanna change hCaptcha side and continue show hCaptcha for some time?

Exactly, I want the hcaptcha to toggle from the compact size to the challenge opened size.
